### PR TITLE
[FIX] test_pylint: Fix empty paths

### PR DESCRIPTION
**Pull request by moylop260**
This fix is to avoid next issue:
https://github.com/OCA/runbot-addons/pull/53

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1:

**Comment by nhomar** [comment]
:+1:

**Comment by moylop260** [comment]
@lepistone 
Can you review this Pr, please?

**Comment by dreispt** [comment]
:+1:

**Comment by pedrobaeza** [comment]
Fixes https://github.com/OCA/maintainer-quality-tools/issues/240.

I merge.

File: travis/run_pylint.py [hunk 1]
```diff
@@ -84,10 +87,13 @@ def main(paths, config_file, sys_paths=None, extra_params=None):
...
-        list(paths), config_file.name, sys_paths=sys_paths,
-        extra_params=extra_params)
-    count_fails = get_count_fails(stats)
+    try:
+        stats = run_pylint(
+            list(paths), config_file.name, sys_paths=sys_paths,
+            extra_params=extra_params)
+        count_fails = get_count_fails(stats)
+    except ValueError:
+        count_fails = -1
```
Comment (dreispt) [inline travis/run_pylint.py]: this has to be `count_fails = 0`.
See https://travis-ci.org/OCA/contract/jobs/77930562

Comment (pedrobaeza) [inline travis/run_pylint.py]: Is still failing?

**Comment by dreispt** [comment]
======== Testing test_pylint ========
    pylint expected errors 0, found -1!

File: travis/run_pylint.py (hunk 1 above)
Comment (dreispt) [inline travis/run_pylint.py]: ======== Testing test_pylint ========
    pylint expected errors 0, found -1!

Comment (pedrobaeza) [inline travis/run_pylint.py]: I have amended the commit to put 0. Rerun Travis.

Comment (pedrobaeza) [inline travis/run_pylint.py]: It's not that simple. I have to reverted again and check carefully

Comment (moylop260) [inline travis/run_pylint.py]: Let me try to fix it

Comment (moylop260) [inline travis/run_pylint.py]: Fixed here:
https://github.com/OCA/maintainer-quality-tools/pull/241

Comment (moylop260) [inline travis/run_pylint.py]: FYI I tested with travis2docker with contract/8.0

**Pull request merged**